dashboard/lib/projects_list.rb

```diff
@@ -125,6 +126,7 @@ def fetch_published_project_types(project_types, limit:, published_before: nil)
     # storage_apps and user tables. See project_and_user_fields for which
     # fields it contains.
     def get_published_project_and_user_data(project_and_user)
```

Comment:
Though I know it's outside of this PR, this method description is unhelpful. Please follow the [YardDoc format](https://www.yardoc.org/) and explain what this method does (with a verb), what it expects as its `@param` and what it `@returns`. This is especially important now that it is expected to return nil in certain cases.